pkg/k8s/endpoints.go

```diff
@@ -94,6 +94,7 @@ func (in *Endpoints) DeepCopy() *Endpoints {
 type Backend struct {
 	Ports         serviceStore.PortConfiguration
 	NodeName      string
+	Hostname      string
```

Comment:
Let's use same style with `NodeName`: `HostName`

Reply:
Thanks for the review! I actually did that to conform to the same style as go struct in Kubernetes that does do the same (`NodeName` + `Hostname`, https://github.com/cilium/cilium/blob/095f4b2ae61ca8c9b434f746ac54e6740fb999e0/pkg/k8s/slim/k8s/api/discovery/v1/types.go#L83), yes it's a bit weird I was confused too :sweat_smile:. If you still think that I need to change it I can do it :+1:

Reply:
Hi @joamaki, fyi I am re-requesting review in case the added context above slipped through (sorry if that's not the case!) about `Hostname` vs `HostName`.